packages/api/tsconfig.json

```diff
@@ -12,7 +12,8 @@
     "noUnusedParameters": true,
     "noUnusedLocals": true,
     "resolveJsonModule": true,
-    "declaration": true
+    "declaration": true,
+    "skipLibCheck": true
```

Comment:
Why do we need this?

Reply:
Because `@types/bun` has some conflicts with `@types/node`

Also i guess we no need to check libraries difinitions

because it already type-checked by library authors